ptf/tests/sai_t0_case.py

```diff
@@ -23,3 +31,50 @@ def tearDown(self):
         Test the basic tearDown proecss
         """
         pass
+
+class L2FdbForwardingTest(SaiT0HelperBase):
```

Comment:
we can use a method name that can specify the purpose more specifically.
There will be different test classes for different tests.
This is a sanity test for checking the configuration?